src/main/webapp/style.css

```diff
@@ -10,42 +10,172 @@ img {
 
 /* Header properties */
```

Comment:
Can you please add this thing?

* {
croll-behavior: smooth;
}

It will make the transition not so abrupt.

Reply:
Done